internal/cmd/iso/list.go

```diff
@@ -23,11 +26,29 @@ var ListCmd = base.ListCmd{
 		cmd.RegisterFlagCompletionFunc("architecture", cmpl.SuggestCandidates(string(hcloud.ArchitectureX86), string(hcloud.ArchitectureARM)))
 
 		cmd.Flags().Bool("include-architecture-wildcard", false, "Include ISOs with unknown architecture, only required if you want so show custom ISOs and still filter for architecture.")
+
+		cmd.Flags().StringSlice("type", []string{"public", "private"}, "Types to include")
```

Comment:
Please indicate the available options as well as the default to the user in the help message.